specs/gluster_object_detail_views.adoc

```diff
@@ -0,0 +1,198 @@
+// vim: tw=79
+:imagesdir: ./images/
+
+= Tendrl Definitions: Display list of volume and host associated with a gluster cluster discovered by Tendrl
+
+== Problem description
+
+Display list of volume and host associated with a gluster cluster present in Tendrl. 
+
+*Host List*
+
+The details shown for each host are -
+
+* Status
+* Name
+* Role
+* Alerts
+
+The host list should have functionality for 
+
+* filtering by name, role and status.
+* sorting by name and role.
+
+Host list will be sorted by name by default.
+
+View design will look like:
+
+image::hostList.png[Gluster specific host list view]
+
+*Volume List*
+
+The details shown for each volume are -
+
+* Status
+* Name
+* Type
+* Bricks
+* Alerts
+* Running: Yes/No
+* Last Rebalance
+
+The volume list should have functionality for 
+
+* filtering by name, type and status.
+* sorting by name and status.
+
+Volume list will be sorted by name by default.
+
+View design will look like:
+
+image::volumeList.png[Gluster specific host list view]
+
+The workflow is as follows:
+
+* API will provide the list of volume and host asscociated with a gluster cluster to the Tendrl.
+
+* User will click on *Cluster Details" button from cluster list view to go to object details view.
+
+* Two tabs will be displayed on the page: Hosts and Volumes
+
+* Clicking hosts tab will list down the hosts associated with that gluster cluster.
+
+* Clicking volumes tab will list down the volumes associated with that gluster cluster.
+
+== Use Casess
+
+Display list of volume and host associated with a gluster cluster present in Tendrl. 
+
+== Proposed change
+
+* Create a view for cluster object detail which will have two tab: Hosts and Volumes.
+* Create components for hosts and volumes list page.
+* Implement client side filer and sorting functionality using angular's filter and orderby components.
+* Send polling call after 10 seconds for volume list and 30 seconds for host list to get the updated data. 
```

Comment:
30 sec polling for hosts of a cluster which is already imported might not be required as this data doesnt change frequently. This could a minute or so. @Tendrl/tendrl-core comments?